File: app/src/main/java/moe/tlaster/openween/core/model/favor/FavorListModel.java
package moe.tlaster.openween.core.model.favor;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Created by Tlaster on 2016/9/2.
 */
public class FavorListModel {
    @SerializedName("favorites")
    private List<FavorModel> mFavorites;
    @SerializedName("total_number")
    private int mTotalNumber;

    public List<FavorModel> getFavorites() {
        return mFavorites;
    }

    public void setFavorites(List<FavorModel> favorites) {
        mFavorites = favorites;
    }

    public int getTotalNumber() {
        return mTotalNumber;
    }

    public void setTotalNumber(int totalNumber) {
        mTotalNumber = totalNumber;
    }
}

File: app/src/main/java/moe/tlaster/openween/core/model/favor/FavorListModel.kt
package moe.tlaster.openween.core.model.favor

import com.fasterxml.jackson.annotation.JsonProperty


/**
 * Created by Tlaster on 2016/9/2.
 */
class FavorListModel {
    @field:JsonProperty("favorites")
    var favorites: List<FavorModel>? = null
    @field:JsonProperty("total_number")
    var totalNumber: Int = 0
}
